acir/src/circuit/opcodes/black_box_function_call.rs

```diff
@@ -62,6 +62,7 @@ pub enum BlackBoxFuncCall {
     },
     Pedersen {
         inputs: Vec<FunctionInput>,
+        hash_index: u32,
```

Comment:
Perhaps this is better called as `domain_separator`? Since it is an index that will be used to separate different hash calls